### PR TITLE
Flattened readme

### PR DESCRIPTION
**Pull request by danjoa**

### CODE REVIEW

**Review by mnkiefer — CHANGES_REQUESTED** [review]
Hi Daniel, looks great! 👍 I like that you now have the *calesi* alternative if you want to skip the step-by-step and jump right in.

One thing though, I believe we still need the header `##About this project` to ensure OSS compliance. @sjvans: Could you please comment on the structure that was requested for the README which you also [adhered to in @cap-js/audit-logging](https://github.com/cap-js/audit-logging/pull/15/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)?

**Comment by sjvans** [comment]
> Hi Daniel, looks great! 👍 I like that you now have the _calesi_ alternative if you want to skip the step-by-step and jump right in.
> 
> One thing though, I believe we still need the header `##About this project` to ensure OSS compliance.
> 
> @sjvans: Could you please comment on the structure that was requested for the README which you also [adhered to in @cap-js/audit-logging](https://github.com/cap-js/audit-logging/pull/15/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)?

```
The README.md file should be broken down into the following sections.
- Mandatory Sections
  - Description
  - Requirements
  - Contributing
  - Code of Conduct
  - Licensing
- Optional Sections
  - Download & Installation
  - Configuration
  - Limitations
  - Known Issues
  - How to Obtain Support
  - To-Do (Upcoming changes)
```

i'll send you some internal links via chat ;)

**Review by sjvans — CHANGES_REQUESTED** [review]
i'm afraid we'll need to retain `.reuse` and `LICENSES`

File: README.md [hunk 1]
```diff
@@ -1,23 +1,55 @@
+# Change Tracking Plugin for SAP Cloud Application Programming Model (CAP)
+
+The `@cap-js/change-tracking` package is a [CDS plugin](https://cap.cloud.sap/docs/node.js/cds-plugins#cds-plugin-packages) providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities as simple as that:
```
Comment (sjvans) [inline README.md:3]:
```suggestion
[![REUSE status](https://api.reuse.software/badge/github.com/cap-js/change-tracking)](https://api.reuse.software/info/github.com/cap-js/change-tracking)

The `@cap-js/change-tracking` package is a [CDS plugin](https://cap.cloud.sap/docs/node.js/cds-plugins#cds-plugin-packages) providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities as simple as that:
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,55 @@
+# Change Tracking Plugin for SAP Cloud Application Programming Model (CAP)
+
```
Comment (sjvans) [inline README.md:2]:
```suggestion

[![REUSE status](https://api.reuse.software/badge/github.com/cap-js/change-tracking)](https://api.reuse.software/info/github.com/cap-js/change-tracking)

```

**Comment by mnkiefer** [comment]
> > Hi Daniel, looks great! 👍 I like that you now have the _calesi_ alternative if you want to skip the step-by-step and jump right in.
> > One thing though, I believe we still need the header `##About this project` to ensure OSS compliance.
> > @sjvans: Could you please comment on the structure that was requested for the README which you also [adhered to in @cap-js/audit-logging](https://github.com/cap-js/audit-logging/pull/15/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)?
> 
> ```
> The README.md file should be broken down into the following sections.
> - Mandatory Sections
>   - Description
>   - Requirements
>   - Contributing
>   - Code of Conduct
>   - Licensing
> - Optional Sections
>   - Download & Installation
>   - Configuration
>   - Limitations
>   - Known Issues
>   - How to Obtain Support
>   - To-Do (Upcoming changes)
> ```
> 
> i'll send you some internal links via chat ;)

@sjvans: Thanks for the links! 👍 According to the repo linting tool, we should still have a section called `Requirements`, right?

> [!WARNING]  
> [rl-readme_file-2] README is incomplete. Please check the following sections: Requirements
Warning

**Comment by sjvans** [comment]
> @sjvans: Thanks for the links! 👍 According to the repo linting tool, we should still have a section called `Requirements`, right?
> 
> Warning [rl-readme_file-2] README is incomplete. Please check the following sections: Requirements Warning

not a must. however, the tool only lints main, so you'll need to merge this first to see if there are new violations.

**Comment by danjoa** [comment]
Let's discuss that in our next sync. 
Priority is conciseness and comprehensibility, not formalism

**Comment by nkaputnik** [comment]
We should also add a disclaimer on the top that this is ****Early Adopters**** and production use is on your own risk.
Nils Hirsekorn already raised a performance isse and "expects" a resolution. Clearly, have diverging expectations from an open source project...

**Review by mnkiefer — APPROVED** [review]

**Comment by mnkiefer** [comment]
> We should also add a disclaimer on the top that this is **Early Adopters** and production use is on your own risk. Nils Hirsekorn already raised a performance isse and "expects" a resolution. Clearly, have diverging expectations from an open source project...

We've addressed this [here](https://github.tools.sap/cap/docs/pull/407) 👍

File: package.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
   "description": "CDS plugin providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities.",
   "repository": "cap-js/change-tracking",
   "author": "SAP SE (https://www.sap.com)",
-  "homepage": "https://cap.cloud.sap/",
```
Comment (sjvans) [inline package.json:7]: why remove?

File: README.md [hunk 4]
```diff
@@ -1,23 +1,57 @@
...
+```sh
+git clone https://github.com/cap-js/incidents-app
+cd incidents-app
+npm i
+```
+
+**Alternatively**, you can clone the incidents app including the prepared enhancements for change-tracking:
+
+```sh
+git clone https://github.com/cap-js/calesi --recursive
```
Comment (sjvans) [inline README.md:44]: not public yet

**Review by sjvans — APPROVED** [review]
please see comments

File: package.json (hunk 3 above)
Comment (mnkiefer) [reply package.json:7]: I also think this should stay as is for now. 

@danjoa: Let's keep it in and we can later on directly link to the envisioned CDS plugin landing page?

File: README.md (hunk 4 above)
Comment (mnkiefer) [reply README.md:44]: @danjoa: Let's take this out until made public?

Comment (sjvans) [reply README.md:44]: or we make it public now ;)

Comment (danjoa) [reply README.md:44]: cap-js/calesi is public now

File: package.json (hunk 3 above)
Comment (danjoa) [reply package.json:7]: I think default is github readme which is better than cap.cloud.sap in this case, as it is a plugin.

**Pull request merged**